src/test/crimson/perf_crimson_msgr.cc

```diff
@@ -176,42 +183,93 @@ static seastar::future<> run(
         : public ceph::net::Dispatcher,
           public seastar::peering_sharded_service<Client> {
 
-      struct PingSession : public seastar::enable_shared_from_this<PingSession> {
+      struct ConnSession {
```

Comment:
IMHO, it's not a session. it's just statistics of a given connection.

Reply:
changed to `ConnStats`